### ISSUE TITLE
0.2.1 version let the webview's footer overlay some part of space (only in iOS)

### ISSUE THREAD

**Issue by bau720123** [Telerik-Verified-Plugins/NativePageTransitions#15]
hi @EddyVerbruggen 
I am using 0.2.1 version with PGB 3.6.3 and iOS 8.1 (Android and WP all work perfect)
https://build.phonegap.com/plugins/1267

my code completely like yours
http://plugins.telerik.com/plugin/native-page-transitions

before animation
http://www.littlebau.com/20141122/01.PNG
after animation
http://www.littlebau.com/20141122/02.PNG

but finally I found a Temporary type of solution
it is use the 3rd status plugin
https://build.phonegap.com/plugins/715
https://github.com/apache/cordova-plugin-statusbar/blob/bdbaf129595c4a4e3d957a1bc218d05dc5f82a9e/README.md

and use these javascript code again

```
StatusBar.overlaysWebView(true);
StatusBar.overlaysWebView(false);
```

after it，the footer become normal again

**Comment by EddyVerbruggen** [comment]
Fixed in 0.2.11, also see #30, thanks for reporting!

**Comment by bau720123** [comment]
according to this
https://github.com/Telerik-Verified-Plugins/NativePageTransitions/issues/30
problem solved
thanks @EddyVerbruggen and @blefebvre
